index.html

```diff
@@ -2,8 +2,12 @@
 <html>
 	<head>
 		<title>Omnichess</title>
+		<script src="src/Component.js"></script>
```

Comment:
I would suggest moving these to the end of the `<body>` section. This should improve load times as elements can render before the scripts need to be loaded. Otherwise, loading these scripts will prevent the browser from loading everything.